spec/util/miq_file_storage_spec.rb

```diff
@@ -0,0 +1,380 @@
+require "pty" # required inline in module, needed here for stubbing
```

Comment:
Do we still need this?

Reply:
Oh, good catch.  Nope, not needed now that we are just using `File.mkfifo`.  Will remove.